vars/runStackPublishPipeline.groovy

```diff
@@ -42,8 +42,8 @@ def call() {
 
                     sh """
                           STACK_VERSION=${params.version}
-                          STACK_VERSION=\${STACK_VERSION} ./bin/kubectl-crossplane-stack-build
-                          STACK_VERSION=\${STACK_VERSION} ./bin/kubectl-crossplane-stack-publish
+                          STACK_VERSION=\${STACK_VERSION} ./bin/kubectl-crossplane-package-build
```

Comment:
i think this env var needs to change also, since builds for some repos seem to be changing to use `PACKAGE_VERSION`, e.g.:
https://github.com/crossplane/stack-gcp-sample/pull/44#discussion_r416188329

Reply:
Ah yes good catch!